package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "restaurant",
+  "version": "1.0.0",
+  "description": "Practising modules in JS",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint",
+    "build": "webpack"
```

Comment:
[OPTIONAL]
Consider also having `development` and `start` scripts.
The `start` script would likely use `webpack-dev-server`